doc/release_notes/v2.0.0.rst

```diff
@@ -8,6 +8,11 @@ Changelog
   Classes `PersonNameUnicode` and `PersonName3` are aliased to `PersonName` but
   are deprecated and will be removed in version 2.1
 
+Enhancements
+............
+* Allow PathLike objects for filename argument in `dcmread` and `dcmwrite`
+  (:issue:`1047`)
+
```

Comment:
Might be good to mention `Dataset.save_as` here - some might not use `dcmwrite` at all

Reply:
Good point!